cider/exec/module/batch/CiderBatchUtils.cpp

```diff
@@ -57,14 +57,13 @@ namespace CiderBatchUtils {
     std::vector<bool> null_data;                                              \
     col_data.reserve(table_row_num);                                          \
     null_data.reserve(table_row_num);                                         \
-    null_data.push_back(false);                                               \
     C_TYPE* buf = (C_TYPE*)table_ptr[i];                                      \
     C_TYPE* null_buf = (C_TYPE*)table_ptr[i * 2];                             \
     for (auto j = 0; j < table_row_num; ++j) {                                \
       C_TYPE value = buf[j];                                                  \
-      bool is_null = null_buf[j];                                             \
+      bool is_not_null = null_buf[j];                                         \
```

Comment:
in this macro, C_TYPE can't be `bool`
so this line should be: `bool is_not_null = (null_buf[j] != 0);` ?

Reply:
Done.